doc/news/_preparation_next_release.md

```diff
@@ -189,7 +189,7 @@ you up to date with the multi-language support provided by Elektra.
 
 ## Tools
 
-- [elektrad](../../src/tools/elektrad/README.md) is completely rewritten in Go - which drastically improves the performance by leveraging the new [go-elektra](https://github.com/ElektraInitiative/go-elektra/) bindings instead of calling the `kdb` command-line tool on every request. _(Raphael Gruber)_
+- [elektrad](../../src/tools/elektrad/README.md) is completely rewritten in Go, which drastically improves the performance by leveraging the new [go-elektra](https://github.com/ElektraInitiative/go-elektra/) bindings instead of calling the `kdb` command-line tool on every request. _(Raphael Gruber)_
```

Comment:
A longer text here would be nice, maybe even a highlight. E.g. how much was the improvement, links to the installation tutorial and so on.

Reply:
Sure, but really isn't in the scope of this PR, right?

Reply:
Of course you can make separate PRs if you prefer. For me it is easier to constantly extend text and not to write a longer text from scratch, that is why I gave you the hint :heart: